src/python/database.py

```
@@ -17,12 +26,22 @@
 
-        Args:
-            db_file (str): The path to the SQLite database file. Defaults to "todo.db" if not provided.
-
-        Raises:
-            None
-        """
+          Raises:
+              None
+          """
+        if db_file is None:
+            DEFAULT_DB_FILE = 'todo.db'
```

Ruff (0.7.0)

Local variable `DEFAULT_DB_FILE` is assigned to but never used

Remove assignment to unused variable `DEFAULT_DB_FILE`

(F841)